src/coreComponents/dataRepository/Group.cpp

```diff
@@ -17,8 +17,13 @@
 #include "ConduitRestart.hpp"
 #include "codingUtilities/StringUtilities.hpp"
 #include "codingUtilities/Utilities.hpp"
+#include "codingUtilities/StringUtilities.hpp"
+#include "common/MpiWrapper.hpp"
 #include "common/TimingMacros.hpp"
 
+#include <numeric>
+#include <unordered_set>
+
```

Comment:
These changes no longer needed with printing function moved to `Utilities.cpp`?
```suggestion
#include "common/TimingMacros.hpp"
```